cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -482,18 +486,44 @@ def _get_creation_failures(cluster_status, cfn_stack):
 
 
 def _get_details(cfn_stack, verbose):
-    return _get_alarms_details(cfn_stack) if verbose else None
+    if verbose:
+        alarms = _get_alarms_details(cfn_stack)
+        metrics = _get_cluster_health_metrics_details(cfn_stack)
+        stats = _get_stats_details(cfn_stack)
+        return [Detail(alarms=alarms, metrics=metrics, stats=stats)]
```

Comment:
The type of details in `DescribeClusterResponseContent` return by this function should be a single Detail object instead of a list of Detail. Also I suggest to rename Detail to Details here for alignment and less confusion